packages/docs/pages/usage/search/sorting.mdx

```diff
@@ -0,0 +1,79 @@
+# Sorting
+
+To sort, Orama uses the properties defined in the `schema` to know on which properties you want to sort.
+
+```javascript copy
+const db = await create({
+  schema: {
+    title: 'string',
+    year: 'number',
+    isTop: 'boolean',
+    meta: {
+      tag: 'string',
+      rating: 'number',
+      favorite: 'boolean',
+    },
+  }
+})
+const results = await search(db, {
+  term: 'prestige',
+  sort: {
+    property: 'title', // or 'year', 'isTop'
+  }
+})
+```
+
+Orama supports sorting on 'string' and 'number'.
+
+You can also specify nested properties using the '.' notation: `'meta.tag'`, `'meta.rating'` and `'meta.favorite'`
+
+## Reverse order
+
+Orama supports the reverse order specifying the key `order`:
+
+```javascript
+
+const db = await create({
+  schema: {
+    title: 'string',
+    year: 'number',
+    isTop: 'boolean',
+    meta: {
+      tag: 'string',
+      rating: 'number',
+      favorite: 'boolean',
+    },
+  }
+})
+const results = await search(db, {
+  term: 'prestige',
+  sort: {
+    property: 'title', // or 'year', 'isTop'
+    order: 'DESC'
+  }
+})
+```
+
+## Memory optimization
+
+By default, Orama allows the sort on all properties defined in the schema.
+This creates an in-memory sort index for each properties.
+If you want to optimize the memory usage, Orama supports the `sortableDeniedProperties` list.
+
+```javascript
+const db = await create({
+  schema: {
+    title: 'string',
+    year: 'number',
+    isTop: 'boolean',
+    meta: {
+      tag: 'string',
+      rating: 'number',
+      favorite: 'boolean',
+    },
+  },
+  sort: {
```

Comment:
I'm not sure about this. 
It's probably fine now, but we might end up with possible endless list of options in the future.

@micheleriva WDYT about moving to an approach likes this?

```
...
schema: {
  title: 'string',
  year: {type: 'number', sortable: false }
  // ...
},
```

Reply:
At this point, why not move to JsonSchema? Creating a "custom" schema language is useless, and the user can confuse it!
Anyway, I prefer to keep it simple: by default, all are sortable; if you want, you can put the properties inside the denied list.

Reply:
Agreed with @allevo. Orama must be simple to use and customize. You can use it with its out of the box features, and then optimize it when/if needed

Reply:
Ok, no problem for me.